src/main/scala/thurloe/service/ThurloeService.scala

```diff
@@ -16,13 +19,52 @@ class Thurloe extends Actor with MyService {
   // this actor only runs our route, but you could add
   // other things here, like request stream processing
   // or timeout handling
-  def receive = runRoute(myRoute)
+  def receive = runRoute(thurloeRoutes)
 }
 
 
 // this trait defines our service behavior independently from the service actor
+@Api(value="/thurloe", description = "Thurloe service", produces = "application/json", position = 1)
 trait MyService extends HttpService {
 
+  import spray.httpx.SprayJsonSupport.sprayJsonUnmarshaller
+
+  def thurloeRoutes: Route =
+  pathPrefix("thurloe") {
+    path(Segment / "get" / Segment) { (userId, key) =>
```

Comment:
I think the "get" could be implied in the route, so clients would just issue a GET against userId/key